code/datums/status_effects/xeno_buffs.dm

```diff
@@ -101,7 +101,7 @@
 	///Minimum health threshold before the effect is deactivated
 	var/minimum_health
 	///If the target xeno was within range
-	var/was_within_range = TRUE
+	var/was_within_range
```

Comment:
is this a bool or not cus itf its a bool make it false explicitly

Reply:
updated